telegram_packs/logos.txt

```diff
@@ -0,0 +1,24 @@
+logos/frogAlpine|💿
```

Comment:
:penguin: for the linux distros?

Reply:
We already have 2 penguins. I dont think all Linux disteos should map to that.

Reply:
maybe, but IMO a penguin is usually way more associated with linux distros, than a CD.

Reply:
alright
will add the mapping